src/common-principles.md

```diff
@@ -669,7 +669,10 @@ to which the metadata contained within are not applicable.
 The Inheritance Principle defines a systematized set of rules
 to determine which metadata files to associate with which data files.
 Further, because multiple metadata files may apply to an individual data file,
-the Principle defines the *order of precedence* of such metadata files contents.
+the Principle defines the *order of precedence* of such metadata content;
+this is necessary for resolution of conflicts if the same metadata field
+contains different values in different metadata files
+(though it is RECOMMENDED to avoid such overloading).
```

Comment:
since in BIDS 2.0 we make it not allowed (ERROR), we could also relax (completely remove?) the order of precedence statement since it would not matter, right?  or should we leave it in to disambiguate implementations robust operation and say that e.g. top level (most common) definition takes precedence?

Reply:
The mechanisms relating to determination of the order of metadata files remains relevant, not for `.json` but for other metadata extensions. For formats where it is only the "nearest" metadata file that applies, there has to be no ambiguity about which is "nearest".

Reply:
well, while it is just a WARNING it remains relevant even for JSONs .  Even when it would be ERROR, it would be nice to describe expected behavior in case of non-compliance, but I think it would not be possible if we relax requirements and allow multiple files at the same level to provide it.